src/main/java/org/mule/service/scheduler/internal/DefaultSchedulerService.java

```diff
@@ -54,7 +56,11 @@
  */
 public class DefaultSchedulerService implements SchedulerService, Startable, Stoppable {
 
+  private static final String USAGE_TRACE_INTERVAL_SECS_PROPERTY = "mule.scheduler.usageTraceIntervalSecs";
+  public static final Long USAGE_TRACE_INTERVAL_SECS = getLong(USAGE_TRACE_INTERVAL_SECS_PROPERTY);
+
   private static final Logger logger = getLogger(DefaultSchedulerService.class);
+  public static final Logger traceLogger = getLogger("org.mule.service.scheduler.trace");
```

Comment:
`mule.scheduler.trace` is probably enough given the rest is fairly redundant,  Also not using `org` help differentiate it from class-name categories. .  But as we don't have a defined nomenclature for log categories (only what we use for system properties) I won't insist if you think this is best.